specs/HiddenPdfToolbarItems.md

```diff
@@ -0,0 +1,145 @@
+And this PDF viewer is also available in WebView2. And we provide the API to enable users to customization the PDF toolbar.
+
+
+# Description
+We add a new `HiddenPdfToolbarItems` property in `CoreWebView2Settings`. This API allows users to hide buttons on the PDF toolbar. 
+Currently, this API can hide the _save button_, _save as button_, and _print button_.
+By default, `HiddenPdfToolbarItems` is equal to `None` which means no button will be hidden.
+
+# Examples
+## C++
+
+```cpp
+wil::com_ptr<ICoreWebView2> webView;
+void SettingsComponent::ToggleHidePdfToolbarButtons()
+{
+    // Get webView's current settings
+    wil::com_ptr<ICoreWebView2Settings> coreWebView2Settings;
+    CHECK_FAILURE(webView->get_Settings(&coreWebView2Settings));
+    
+    COREWEBVIEW2_PDF_TOOLBAR_ITEM hiddenPdfToolbarItems;
+    CHECK_FAILURE(coreWebView2Settings->get_HiddenPdfToolbarItems(&hiddenPdfToolbarItems));
+    
+    if (hiddenPdfToolbarItems == COREWEBVIEW2_PDF_TOOLBAR_ITEM::COREWEBVIEW2_PDF_TOOLBAR_ITEM_NONE)
+    {
+        CHECK_FAILURE(coreWebView2Settings->put_HiddenPdfToolbarItems(
+            COREWEBVIEW2_PDF_TOOLBAR_ITEM::COREWEBVIEW2_PDF_TOOLBAR_ITEM_PRINT |
+            COREWEBVIEW2_PDF_TOOLBAR_ITEM::COREWEBVIEW2_PDF_TOOLBAR_ITEM_SAVE));
+    }
+    else
+    {
+        CHECK_FAILURE(coreWebView2Settings->put_HiddenPdfToolbarItems(
+            COREWEBVIEW2_PDF_TOOLBAR_ITEM::COREWEBVIEW2_PDF_TOOLBAR_ITEM_NONE));
+    }
+}
+```
+
+## C#
+```c#
+private WebView2 _webView;
+void ToggleHidePdfToolbarButtons(object target, ExecutedRoutedEventArgs e)
+{
+    var coreWebView2Settings = _webView.CoreWebView2.Settings;
+    if(WebViewSettings.HiddenPdfToolbarItems == CoreWebView2PdfToolbarItem.Save | CoreWebView2PdfToolbarItem.Print)
+    {
+        WebViewSettings.HiddenPdfToolbarItems = CoreWebView2PdfToolbarItem.None;
+    }
+    else
+    {
+        WebViewSettings.HiddenPdfToolbarItems = CoreWebView2PdfToolbarItem.Save;
+    }
+}
+```
+
+# API Notes
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+## Win32 C++
+```c#
+/// Specifies the PDF toolbar item types used for the `ICoreWebView2StagingSettings::put_HiddenPdfToolbarItems` method.
+[v1_enum]
+typedef enum COREWEBVIEW2_PDF_TOOLBAR_ITEM {
+
+  /// No item
+
+  COREWEBVIEW2_PDF_TOOLBAR_ITEM_NONE  = 0x0,
+
+  /// The save button
+
+  COREWEBVIEW2_PDF_TOOLBAR_ITEM_SAVE  = 0x0001,
+
+  /// The print button
+
+  COREWEBVIEW2_PDF_TOOLBAR_ITEM_PRINT  = 0x0002,
+
+  /// The save as button
+
+  COREWEBVIEW2_PDF_TOOLBAR_ITEM_SAVE_AS  = 0x0004,
+
+
+} COREWEBVIEW2_PDF_TOOLBAR_ITEM;
+cpp_quote("DEFINE_ENUM_FLAG_OPERATORS(COREWEBVIEW2_PDF_TOOLBAR_ITEM);")
+
+[uuid(183e7052-1d03-43a0-ab99-98e043b66b39), object, pointer_default(unique)]
+interface ICoreWebView2Settings6 : ICoreWebView2Settings5 {
+  /// `HiddenPdfToolbarItems` is used to customize the PDF toolbar items. By default, it displays all of the items.
```

Comment:
If I recall from a different conversation this setting applies to more than just this CoreWebView2, correct? Does it apply to all CoreWebView2s using the same browser process and profile? If so, we need to specify this in the documentation here. 
```suggestion
  /// `HiddenPdfToolbarItems` is used to customize the PDF toolbar items. By default, it displays all of the items.
  /// Changes to this property apply to all CoreWebView2s in the same environment and using the same profile.
```